src/main/java/me/ryanhamshire/GriefPrevention/util/BoundingBox.java

```diff
@@ -0,0 +1,732 @@
+package me.ryanhamshire.GriefPrevention.util;
```

Comment:
I was going for a more future state with this - should this be in the package `com.griefprevention.util` instead? I know you control the domain, just didn't want to rock the boat too too much until breaking change time.

Reply:
well I'm starting to go through PRs now, so change time is coming soon. I don't really see a need to refactor packages, but I guess my plan with that is once we "start work" on 17 with new classes I'll put those under the new domain.

Reply:
Gotcha. I ask because it's a new package anyway, but then again, I'd like to eventually make Claim extend it and remove the Claim constructor. I guess it's got breaking changes in the pipe either way.